### PR TITLE
feat(plugins): Prefer plugins from deck manifest (over gate) if they have the same ID

### PR DESCRIPTION
**Pull request by christopherthielen**
If deck has a plugin-manifest which includes a plugin ID that is also served by gate, prefer the one in deck.

This can happen for two reasons:
- Deck has baked in manifest and plugins
- Developer is updating a plugin and is running in dev mode

### CODE REVIEW

**Comment by christopherthielen** [comment]
@bpowell ptal

File: plugin-manifest.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-const plugins = [];
+const plugins = [{ id: 'com.robzienert.helloworld', version: 'snapshot', url: '/plugins/helloworld/index.js' }];
```
Comment (bpowell) [inline plugin-manifest.js]: Will adding this is without the corresponding `index.js` file cause any issues?

Comment (christopherthielen) [reply plugin-manifest.js]: oh wow, nice catch!  That's not supposed to be there.

**Comment by bpowell** [comment]
@christopherthielen is there something blocking this that I can help unblock?

**Comment by christopherthielen** [comment]
@bpowell no, I was just waiting for an oss-approver to thumbs up this, but I'll just self merge

**Pull request merged**